osu.Desktop/Program.cs

```diff
@@ -102,7 +102,7 @@ public static void Main(string[] args)
                 }
             }
 
-            using (DesktopGameHost host = Host.GetSuitableDesktopHost(gameName, new HostOptions { BindIPC = !tournamentClient }))
+            using (DesktopGameHost host = Host.GetSuitableDesktopHost(gameName, new HostOptions { IPCPort = !tournamentClient ? OsuGame.IPC_PORT : null }))
```

Comment:
Can we give debug builds a different IPC port so one can run a release version of lazer while a debug version is already running? That has been bothering me for quite some time now.

Reply:
I guess yeah

Reply:
> so one can run a release version of lazer while a debug version is already running

I don't ever recall needing to do that though? What's the rationale?

Reply:
Often times I would be on a local version of lazer and then notice something weird. Fastest way for me to check it's not my fault or it's not a regression would be to fire up a release instance and check if it happens there.